CHANGELOG.md

```diff
@@ -32,6 +32,7 @@
 - `[jest-config]` Add `dependencyExtractor` option to use a custom module to extract dependencies from files ([#7313](https://github.com/facebook/jest/pull/7313), [#7349](https://github.com/facebook/jest/pull/7349), [#7350](https://github.com/facebook/jest/pull/7350))
 - `[jest-haste-map]` Accept a `getCacheKey` method in `hasteImplModulePath` modules to reset the cache when the logic changes ([#7350](https://github.com/facebook/jest/pull/7350))
 - `[jest-config]` Add `haste.computeSha1` option to compute the sha-1 of the files in the haste map ([#7345](https://github.com/facebook/jest/pull/7345))
+- `[expect]` `expect(Infinity).toBeCloseTo(Infinity)` works as intended aswell as with `_Infinity` ([#7405](https://github.com/facebook/jest/pull/7405))
```

Comment:
It think there's an extra `_` before the infinity here

Also the description might be easier to understand as "Treat Infinity as equal in toBeCloseTo matcher", sorry I didn't read it during the last review

Reply:
Much clearer and shorter. Updating with your suggestion.